Modules/User/Resources/views/index.blade.php

```diff
@@ -4,7 +4,9 @@
 <div class="container" id="vueContainer">
     @includeWhen(session('success'), 'toast', ['message' => session('success')])
 	@include('user::layouts.navbar')
+	<div class="ml-3">
 	<h4>User Management</h4>
+	</div>
```

Comment:
Can we write like this? We don't need to extra `div`.
```suggestion
	<h4 class="ml-3">User Management</h4>
```